docs/pages/indexer.mdx

```diff
@@ -47,14 +47,131 @@ They are written under the assumption you are using `anvil` for your test chain,
 
 ### Using the indexer with TypeScript
 
+You can see an example of an indexer client [here](https://github.com/latticexyz/mud/tree/230810-indexer-ts/examples/indexer-client).
```

Comment:
```suggestion
You can see [an example of an indexer client in the MUD repo](https://github.com/latticexyz/mud/tree/main/examples/indexer-client).
```

two changes here:
- updates the link to the main branch location once this is merged
- generally should avoid using "here" as link text, so updates to link to wrap the subject we're pointing at/providing more info abot